docs/en/siem/siem-ui.asciidoc

```diff
@@ -256,6 +278,25 @@ It shows relevant contextual information that helps tell the backstory of the
 event. If you see a particular item that interests you, you can drag it to the
 drop area for further introspection.
 
+[float]
+[[import-export-timelines]]
+==== Export and import timelines
+
+You can import and export timelines, which enables importing timelines from one
+{kib} space or instance to another. Exported timelines are saved in an
+http://ndjson.org[`ndjson`] file.
+
+. Go to *SIEM* -> *Timelines*.
+. To export timelines, do one of the following:
+
+* To export one timeline, click the more actions icon in the relevant row and
+then select _Export selected_.
+* To export multiple timelines, select all the required timelines and then click
+*Batch actions* -> _Export selected_.
```

Comment:
Consider changing `Batch actions` to `Bulk actions`

Reply:
Thanks